ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-main.yaml

```diff
@@ -70,17 +70,16 @@ tests:
     export XDG_CACHE_HOME="/tmp/cache"
     export GOCACHE="/tmp/cache"
     export GOMODCACHE="/tmp/cache"
+    export GITHUB_TOKEN_PATH=/creds-github/token
     ./tools/automator-main.sh \
     --org=openshift-service-mesh \
     --repo=sail-operator \
     --branch=main \
     --token-path=/creds-github/token \
     '--title=Automator: merge upstream changes to $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH' \
-    "--labels=tide/merge-method-merge,auto-merge" \
+    "--labels=auto-merge" \
     --modifier=merge_upstream_main \
-    --merge-repository=https://github.com/istio-ecosystem/sail-operator \
-    --merge-branch=main \
-    --merge-strategy=merge
+    --cmd=./ossm/merge_upstream.sh
```

Comment:
Maybe `--script-path` would be better here? Instead of `--cmd`

Reply:
Oh, you're right, I actually wanted to use that

Reply:
turns out I can't use --script-path as it immediately calls `realpath <script-path>` before checking out the repo containing the script